runner/run_benchmark.py

```diff
@@ -138,10 +140,13 @@ def check_result_files(run_name: str, agent_name: str, force_removal: bool = Fal
                     default_yes=False,
                 ):
                     colour_print("red", "Run aborted.")
-                    return
+                    exit()
```

Comment:
Why is this an exit now? Seems too drastic.

Reply:
I have no idea how it worked before. If you choose not to continue, the run continues anyway. The run actually aborts now.

Reply:
And what about raising an exception instead? I personally try not to use `exit` unless it's a quick script.